docs/data/date-pickers/getting-started/getting-started.md

```diff
@@ -121,16 +262,6 @@ You don't have to import the theme augmentation from both `@mui/x-date-pickers`
 Importing it from `@mui/x-date-pickers-pro` is enough.
 :::
 
-## Native pickers
```

Comment:
Agreed, IMHO this section made little to no sense in this location...
Maybe it would have usefulness at the beginning of the page, but that is also debatable.
If people are in search of a date library, they probably already know that native solutions lack what they need. 🤷 🙈 💡